scripts/release.sh

```diff
@@ -0,0 +1,281 @@
+fi
+
+
+
+# 3 update RPC versions
+# /SmartDeviceLink/private/SDLGlobals.m
+# TODO - extract version, prompt user to fix it if necessary.
+echo 
+echo "Step 3: Please update RPC version in /SmartDeviceLink/private/SDLGlobals.h"
+
+# 3 update protocol versions
+# /SmartDeviceLink/private/SDLGlobals.m
+# TODO - extract version, prompt user to fix it if necessary.
+echo 
+echo "Step 3: Please update protocol versions in /SmartDeviceLink/private/SDLGlobals.h"
+
+
+# 4 Update to the newest BSON submodule. Update Package.swift and CocoaPods dependency files to point to latest if necessary.
+# extract version and link from Package.swift
+submodule_info=$(sed -n '/.package/{s/let package = Package(//;s/.package(//;s/)//;p;}' $project_file)
+submodule_name=$(jq -n "{$submodule_info}" | jq -r .name)
+submodule_url=$(jq -n "{$submodule_info}" | jq -r .url)
+submodule_current_version=$(jq -n "{$submodule_info}" | jq -r .from)
+
+# figure out latest version (visit link?)
+submodule_latest_version=$(gh repo view $submodule_url --json latestRelease -q .latestRelease.tagName)
+
+# compare versions
+if [ $submodule_current_version != $submodule_latest_version ]; then
+    echo
+    echo "Current version of $submodule_name: "$submodule_current_version
+    echo "Latest version of $submodule_name: "$submodule_latest_version
+    echo "Please update the submodule $submodule_name before continuing with the release."
+    echo
+    echo "You must also edit the dependancy information in SmartDeviceLink-iOS.podspec"
+    echo
+fi
+
+
+# 5 update changelog
+# IDEA - I should record the timestamp of the changelog, and then check after the user returns to see that they did touch the file.
+# IDEA - we could also insert a template into the changelog that includes the version the users have selected above.
+echo 
+echo "Please go update Changelog.md, then return here and press enter."
+read user_input
+# Changelog.md
+
+# 6 generate documentation
+echo 
+echo "6 generate documentation"
+echo "Would you like to automatically generate documentation with Jazzy (Y/n)?"
+read user_input
+if [[ ! $user_input == [Nn] ]]; then    
+    # 6 Install Jazzy
+    # Check if Jazzy is already installed, and if not then install jazzy
+    echo "Check if Jazzy is already installed, and install if necessary (may require password)"
+    if [ -z "$(mdfind -name 'Jazzy')" ]; then
+        echo "6 Install Jazzy"
+        sudo gem install jazzy
+    fi
+
+    # this runs Jazzy to generate the documentation
+    echo "Run Jazzy to generate documentation"
+    jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+#else
+#    echo "not Jazzy..."
+fi
+
+# 7 ensure RPC_SPEC has released to master
+echo
+echo "Step 7: ensure RPC_SPEC has released to master"
+# TODO - cleanup wording
+# IDEA - maybe provide links to assist?
+echo "Please ensure RPC_SPEC has released to master and return here"
```

Comment:
```suggestion
echo "Check if there's a new release of the RPC_SPEC to master. If there is, please update the rpc_spec submodule to point to the new master, then press enter."
```

Alt: If you have the link, open it for the user.

Reply:
I combined these ideas because I realized we need to give the instructions before opening the page.
Also, no contractions.